logs.go

```diff
@@ -42,6 +42,24 @@ var logsCommand = &cli.Command{
 			Aliases: []string{"f"},
 			Usage:   "Follow log output",
 		},
+		&cli.BoolFlag{
+			Name:    "timestamps",
+			Aliases: []string{"t"},
+			Usage:   "Show timestamps",
+		},
+		&cli.StringFlag{
+			Name:    "tail",
+			Aliases: []string{"n"},
+			Usage:   "Number of lines to show from the end of the logs (default all)",
```

Comment:
Set `cli.StringFlag.Value` to "all", and remove "(default all)" from `cli.StringFlag.Usage` string.